config-service-impl/build.gradle.kts

```diff
@@ -22,6 +22,12 @@ dependencies {
   testImplementation("org.junit.jupiter:junit-jupiter:5.5.2")
   testImplementation("org.mockito:mockito-core:3.3.3")
   testImplementation("org.hypertrace.core.grpcutils:grpc-client-utils:0.3.3")
+
+  constraints {
+    runtimeOnly("org.mongodb:mongodb-driver-sync:4.1.2") {
```

Comment:
Likewise, this comes from document-store - I see it came up on https://github.com/hypertrace/document-store/pull/37 @kotharironak  - any reason it wasn't updated? Can we get a PR in to fix it since it's leaking out to all consumers

Reply:
Will raise this pr.